charts/document-engine/values.yaml

```diff
@@ -10,91 +10,79 @@ image:
 imagePullSecrets: []
 nameOverride: ""
 fullnameOverride: ""
+
 #
 # https://pspdfkit.com/guides/document-engine/configuration/overview/
 # https://pspdfkit.com/guides/document-engine/deployment/product-activation/
 #
-pspdfkit:
-  license:
-    # Activation key for online activation (most common) or
-    # license key for offline activation.
-    # Results in `ACTIVATION_KEY` environment variable
-    # or `LICENSE_KEY` (if `isOffline` is `true`)
-    activationKey: ""
-    # Is the license offline: set to `true` if
-    # an offline license key was provided in `activationKey`
-    isOffline: false
-    # Query existing secret for the activation key
-    externalSecret:
-      # External secret name for license
-      name: ""
-      # The key in the secret used to retrieve the activation key
-      key: DOCUMENT_ENGINE_ACTIVATION_KEY
-  auth:
-    api:
-      # `API_AUTH_TOKEN` should be long enough
-      apiToken: secret
-      # JSON Web Tokens
-      jwt:
-        # Enable JWT
-        enabled: false
-        # `JWT_PUBLIC_KEY`
-        publicKey: none
-        # `JWT_ALGORITHM`
-        # Supported algorithms: RS256, RS512, ES256, ES512.
-        # See RFC 7518 for details about specific algorithms.
-        algorithm: RS256
-      # Use an external secret for API credentials
-      externalSecret:
-        # External secret name
-        name: ""
-        # Key names
-        apiTokenKey: API_AUTH_TOKEN
-        jwtPublicKeyKey: JWT_PUBLIC_KEY
-        jwtAlgorithmKey: JWT_ALGORITHM
-    # Document Engine Dashboard settings
-    dashboard:
-      # Enable dashboard
-      enabled: true
-      # `DASHBOARD_USERNAME`
-      username: admin
-      # `DASHBOARD_PASSWORD` — will generate a random password if not set
-      password: ""
-      # Use an external secret for dashboard credentials
-      # instead of the values from `pspdfkit.auth.dashboard.*`
-      externalSecret:
-        # External secret name
-        name: ""
-        # Key names
-        usernameKey: DASHBOARD_USERNAME
-        passwordKey: DASHBOARD_PASSWORD
+documentEngineLicense:
+  # Activation key for online activation (most common) or
+  # license key for offline activation.
+  # Results in `ACTIVATION_KEY` environment variable
+  # or `LICENSE_KEY` (if `isOffline` is `true`)
+  activationKey: ""
+  # Is the license offline: set to `true` if
+  # an offline license key was provided in `activationKey`
+  isOffline: false
```

Comment:
You don't need this anymore, you can use `ACTIVATION_KEY` at all times since 1.2.0 https://pspdfkit.com/changelog/document-engine/#1.2.0

Reply:
Addressed:
* https://github.com/PSPDFKit/helm-charts/pull/20/commits/5eb931808a53e3455f781a5911f8a7a23074ece3
* https://github.com/PSPDFKit/PSPDFKit-Website/pull/10352/commits/c2e735196fa66f0d3c843997f0b41771c77a292a